docs/macios/xcsync.md

```diff
@@ -1,15 +1,21 @@
 ---
-title: Xcode sync
-description: Learn how to use Xcsync, to provide .NET developers with improved support for editing Apple specific files.
+title: xcsync
```

Comment:
Thoughts on using `dotnet xcsync` for the title and the examples? If you look at the other dotnet tool docs it is formatted that way. One reason for it is that `dotnet xcsync` will work regardless of installing the tool globally or not.

Reply:
initially I was inclined to say yes, but after revisiting other examples like [dotnet format](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format), I feel like this change could be a bit misleading for users.

format is run as `dotnet format [option]`
whereas here, the tool is run as `xcsync generate [option]` or `xcsync sync [option]` 

so I would prefer to keep as is unless we change how the tool is invoked.

Reply:
> whereas here, the tool is run as xcsync generate [option] or xcsync sync [option]

Again, that is only true if the tool is installed globally. While we recommend it be installed globally, we should not assume that it is.